numpy/_core/src/multiarray/dtypemeta.h

```diff
@@ -291,6 +295,37 @@ PyArray_SETITEM(PyArrayObject *arr, char *itemptr, PyObject *v)
                 Py_XSETREF(descr, _new_);  \
         } while(0)
 
+static inline int
+PyArray_GetSortFunction(PyArray_Descr *descr, 
+    NPY_SORTKIND which, int descending, PyArray_SortFunc **out_sort)
+{
+    if (NPY_DT_SLOTS(NPY_DTYPE(descr))->get_sort_function == NULL) {
+        return -1;
+    }
+
+    NPY_DT_SLOTS(NPY_DTYPE(descr))->get_sort_function(
+        descr, which, descending, out_sort);
+    return 0;
+}
+
+static inline int
+PyArray_GetArgSortFunction(PyArray_Descr *descr, 
+    NPY_SORTKIND which, int descending, PyArray_ArgSortFunc **out_argsort)
+{
+    if (NPY_DT_SLOTS(NPY_DTYPE(descr))->get_argsort_function == NULL) {
+        return -1;
+    }
+
+    NPY_DT_SLOTS(NPY_DTYPE(descr))->get_argsort_function(
+        descr, which, descending, out_argsort);
+    return 0;
+}
+
+static inline PyArray_CompareFunc *
+PyArray_SortCompare(PyArray_Descr *descr)
```

Comment:
I'd call this `PyArray_GetSortCompareFunction`

Reply:
Yes, made this change. Thanks